src/consensus/consensus.h

```diff
@@ -18,12 +18,13 @@ static inline bool BIP102active(bool fSegwitSeasoned)
 }
 
 static const unsigned int MAX_LEGACY_BLOCK_SIZE = (1 * 1000 * 1000);
+static const unsigned int MAX_BASE_BLOCK_SIZE = (2 * 1000 * 1000);
 inline unsigned int MaxBlockBaseSize(bool fSegwitSeasoned)
 {
     if (!BIP102active(fSegwitSeasoned))
         return MAX_LEGACY_BLOCK_SIZE;
 
-    return (2 * 1000 * 1000);
+    return MAX_BASE_BLOCK_SIZE;
```

Comment:
IMO the seeds nodes shouldn't be hard-coded.  We should be able to import a list of them (as per qbittorrent etc.).